R/plumber-step.R

```diff
@@ -130,7 +132,7 @@ PlumberEndpoint <- R6Class(
     params = NA,
     tags = NA,
     canServe = function(req){
-      req$REQUEST_METHOD %in% self$verbs && !is.na(stringi::stri_match(req$PATH_INFO, regex=private$regex$regex)[1,1])
+      req$REQUEST_METHOD %in% self$verbs && !is.na(stri_match_first_regex(req$PATH_INFO, private$regex$regex)[1,1])
     },
```

Comment:
I think previously stringi had an execution time advantage over normal R regex because it dropped directly to C without having a lot of overhead. Moving everything to regular R regex would still work but you would have an execution time hit. Depends on what you want. If you are already importing stringi package anyway, might as well use it everywhere.

Reply:
Sounds good, thanks!